taker-frontend/src/components/CloseButton.tsx

```diff
@@ -27,7 +27,7 @@ interface Props {
 /// maker is online.
 export default function CloseButton({ cfd, request, status, buttonTitle, isForceCloseButton }: Props) {
     const disableCloseButton = cfd.state.getGroup() === StateGroupKey.CLOSED
```

Comment:
This button being enabled or not should really be controlled by the backend, based on whether or not the action is returned. That is the point of a RESTful design, that you don't need to duplicate this logic in the frontend :)

Reply:
It is an absolute anti-pattern to reason about the state in the frontend for what you can and cannot do!

Reply:
yeah, we should clean this up. I did not want to go down the rabbit hole of cleaning up the projection cfd states now 😬
I'll create a ticket

Reply:
https://github.com/itchysats/itchysats/issues/1314